### PR TITLE
[5.5] Use Authenticatable-functionality to update remember_token

### PR DESCRIPTION
**Pull request by Blizzke**
`ResetsPasswords` currently assumes a user has a `remember_token` field. 
This changes this to using the `setRememberToken()`-function. 
True, it still assumes `Authenticatable`, but now you can intercept this and still use the built-in reset password functionality.

### CODE REVIEW

File: src/Illuminate/Foundation/Auth/ResetsPasswords.php [hunk 1]
```diff
@@ -100,10 +100,9 @@ protected function credentials(Request $request)
      */
     protected function resetPassword($user, $password)
     {
-        $user->forceFill([
-            'password' => bcrypt($password),
-            'remember_token' => Str::random(60),
-        ])->save();
+        $user->forceFill(['password' => bcrypt($password)]);
```
Comment (lucasmichot) [inline src/Illuminate/Foundation/Auth/ResetsPasswords.php]: In this case we can avoid to call `forceFill`; `$user->password = bcrypt($password);` is just fine

Comment (Blizzke) [inline src/Illuminate/Foundation/Auth/ResetsPasswords.php]: You are absolutely right, thanks.

**Comment by lucasmichot** [comment]
Bad rebase @Blizzke

**Comment by Blizzke** [comment]
@lucasmichot Sorry, not a PR expert, haven't changed anything except a new commit. What did I do wrong?

**Comment by lucasmichot** [comment]
You've pushed a bit too many commits ;-) 
https://github.com/laravel/framework/pull/19181/files

**Comment by Blizzke** [comment]
Yeah I merged in the upstream changes and just pushed, was added automatically here. Is there an easy way to rectify this or should I just close this one, merge and recommit the entire thing?

**Pull request closed without merging**